lndr-backend/data/lndr-server.config.test

```diff
@@ -66,7 +66,7 @@ aws {
 
 notifications {
     api-key = "bobloblawlobslawbomb"
-    api-url = "https://2nifms5o2h.execute-api.us-east-2.amazonaws.com/test/testNotificationUA"
+    api-url = "https://vpz2hgy54f.execute-api.us-east-2.amazonaws.com/prod/testNotificationUA"
```

Comment:
Beware wiring production services up to automated tests. Mixing up traffic between production and tests is something we should try and avoid.